config/jobs/kubernetes/publishing-bot/publishing-bot-presubmits.yaml

```diff
@@ -0,0 +1,16 @@
+presubmits:
+  kubernetes/publishing-bot:
+  - name: pull-publishing-bot-build
+    always_run: true
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/gcloud-in-go:v20171113-192bec25
+        args:
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--root=/go/src"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - --scenario=execute
+        - --
+        - make
```

Comment:
Ptal. We had no script defined in [`travis.yml`](https://github.com/kubernetes/publishing-bot/blob/master/.travis.yml). By default travis runs `make` if no script is specified so I've updated to just run `make`. And we only need Go for this.